.github/workflows/code-health.yml

```diff
@@ -28,7 +23,45 @@ jobs:
           node-version: 16.x
       - name: Install dependencies
         run: yarn install --check-files
-      - name: build
-        run: npx projen build
-
-
+      - name: compile
+        run: npx projen compile
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: actionlint
+        run: |
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+          ./actionlint -color
+        shell: bash
+      - name: Setup Node.js
+        uses: actions/setup-node@v3
+        with:
+          node-version: 16.x
+      - name: Install dependencies
+        run: yarn install --check-files
+      - name: lint
+        run: npx projen eslint
+  dependency-review:
+    runs-on: ubuntu-latest
+    if: ${{ github.event_name == 'pull_request' }}
+    steps:
+      - name: 'Checkout Repository'
+        uses: actions/checkout@v3
+      - name: 'Dependency Review'
+        uses: actions/dependency-review-action@v3
+  shellcheck:
+    name: shellcheck
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Run ShellCheck
+        uses: bewuethr/shellcheck-action@v2
+  call-package-workflow:
+    needs: [compile, lint, dependency-review, shellcheck]
+    secrets: inherit
+    permissions:
+      contents: write
+      pull-requests: read
+    uses: ./.github/workflows/package.yml
```

Comment:
This calls the package.yaml only if all the jobs in the code-health are successfull